pkg/blackduck/ctl_blackduck.go

```diff
@@ -220,6 +239,9 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringSliceVar(&ctl.ImageRegistries, "image-registries", ctl.ImageRegistries, "List of image registries")
 	cmd.Flags().StringVar(&ctl.ImageUIDMapFilePath, "image-uid-map-file-path", ctl.ImageUIDMapFilePath, "Absolute path to a file containing a map of Container UIDs to Tags")
 	cmd.Flags().StringVar(&ctl.LicenseKey, "license-key", ctl.LicenseKey, "License Key of Black Duck")
+	cmd.Flags().StringVar(&ctl.AdminPassword, "admin-password", ctl.AdminPassword, "password for the 'admin' account in postgres")
```

Comment:
For consistency and to match the external db flag description, I think that Postgres should have an uppercase P.  

@msenmurugan Do we want to say admin / user or the name of the actual users in the db (blackduck/blackduck_user) here?  I don't have any preferences.